media-gfx/gimp/gimp-2.99.6.ebuild

```diff
@@ -0,0 +1,255 @@
+
+EAPI=7
+
+LUA_COMPAT=( luajit )
+PYTHON_COMPAT=( python3_{7..10} )
+GNOME2_EAUTORECONF=yes
+VALA_MIN_API_VERSION="0.40"
+VALA_USE_DEPEND=vapigen
+
+inherit autotools gnome2 lua-single python-single-r1 toolchain-funcs vala virtualx
+
+DESCRIPTION="GNU Image Manipulation Program"
+HOMEPAGE="https://www.gimp.org/"
+SRC_URI="mirror://gimp/v2.99/${P}.tar.bz2"
+LICENSE="GPL-3 LGPL-3"
+SLOT="0/3"
+KEYWORDS=""
+
+IUSE="aalib alsa aqua debug doc gnome heif javascript jpeg2k lua mng openexr postscript python udev unwind vala vector-icons webp wmf xpm cpu_flags_ppc_altivec cpu_flags_x86_mmx cpu_flags_x86_sse"
+REQUIRED_USE="lua? ( ${LUA_REQUIRED_USE} )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+RESTRICT="!test? ( test )"
+
+# media-libs/{babl,gegl} are required to be built with USE="introspection"
+# to fix the compilation checking of /usr/share/gir-1.0/{Babl-0.1gir,Gegl-0.4.gir}
+COMMON_DEPEND="
+	>=app-text/poppler-0.90.1[cairo]
+	>=app-text/poppler-data-0.4.9
+	>=dev-libs/atk-2.34.1
+	>=dev-libs/glib-2.62.6:2
+	>=dev-libs/json-glib-1.4.4
+	dev-libs/libxml2:2
+	dev-libs/libxslt
+	>=gnome-base/librsvg-2.40.21:2
+	>=media-gfx/mypaint-brushes-2.0.2:=
+	>=media-libs/babl-0.1.86[introspection,lcms,vala?]
+	>=media-libs/fontconfig-2.12.6
+	>=media-libs/freetype-2.10.2
+	>=media-libs/gegl-0.4.30:0.4[cairo,introspection,lcms,vala?]
+	>=media-libs/gexiv2-0.10.10
+	>=media-libs/harfbuzz-2.6.5
+	>=media-libs/lcms-2.9:2
+	>=media-libs/libmypaint-1.6.1:=
+	>=media-libs/libpng-1.6.37:0=
+	>=media-libs/tiff-4.1.0:0
+	net-libs/glib-networking[ssl]
+	sys-libs/zlib
+	virtual/jpeg
+	>=x11-libs/cairo-1.16.0
+	>=x11-libs/gdk-pixbuf-2.40.0:2
+	>=x11-libs/gtk+-3.24.16:3
+	x11-libs/libXcursor
+	>=x11-libs/pango-1.44.7
+	aalib? ( media-libs/aalib )
+	alsa? ( >=media-libs/alsa-lib-1.0.0 )
+	aqua? ( >=x11-libs/gtk-mac-integration-2.0.0 )
+	heif? ( >=media-libs/libheif-1.9.1:= )
+	javascript? ( dev-libs/gjs )
+	jpeg2k? ( >=media-libs/openjpeg-2.3.1:2= )
+	lua? (
+		${LUA_DEPS}
+		$(lua_gen_cond_dep '
+			dev-lua/lgi[${LUA_USEDEP}]
+		')
+	)
+	mng? ( media-libs/libmng:= )
+	openexr? ( >=media-libs/openexr-2.3.0:= )
+	postscript? ( app-text/ghostscript-gpl )
+	python? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep '
+			>=dev-python/pygobject-3.0:3[${PYTHON_MULTI_USEDEP}]
+		')
+	)
+	udev? ( >=dev-libs/libgudev-167:= )
+	unwind? ( >=sys-libs/libunwind-1.1.0:= )
+	webp? ( >=media-libs/libwebp-0.6.0:= )
+	wmf? ( >=media-libs/libwmf-0.2.8 )
+	xpm? ( x11-libs/libXpm )
+"
+
+RDEPEND="
+	${COMMON_DEPEND}
+	x11-themes/hicolor-icon-theme
+	gnome? ( gnome-base/gvfs )
+"
+
+DEPEND="
+	${COMMON_DEPEND}
+	>=dev-lang/perl-5.30.3
+	>=dev-libs/appstream-glib-0.7.16
+	dev-util/gdbus-codegen
+	dev-util/gtk-update-icon-cache
+	>=dev-util/intltool-0.51.0
+	sys-apps/findutils
+	>=sys-devel/autoconf-2.54
+	>=sys-devel/automake-1.11
+	>=sys-devel/gettext-0.21
+	>=sys-devel/libtool-2.4.6
+	doc? (
+		>=dev-util/gtk-doc-1.32
+		dev-util/gtk-doc-am
+	)
```

Comment:
gimp 2.99.6 now provides gobject-introspection documentation for plugin developers:
```
	doc? (
		app-text/yelp-tools
		dev-libs/gobject-introspection[doctool]
		>=dev-util/gtk-doc-1.32
		dev-util/gtk-doc-am
	)
```

Reply:
Done.